app/src/main/java/com/framgia/soundclound/data/source/remote/TrackRemoteDataSource.java

```diff
@@ -0,0 +1,78 @@
+package com.framgia.soundclound.data.source.remote;
+
+import com.framgia.soundclound.data.model.Collection;
+import com.framgia.soundclound.data.model.CollectionResult;
+import com.framgia.soundclound.data.model.Track;
+import com.framgia.soundclound.data.source.TrackDataSource;
+import com.framgia.soundclound.util.Constant;
+import com.framgia.soundclound.util.StringUtil;
+import com.google.gson.Gson;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Logger;
+
+/**
+ * Created by Sony on 1/5/2018.
+ */
+
+public class TrackRemoteDataSource implements TrackDataSource {
+
+    private static TrackRemoteDataSource sTrackRemoteDataSource;
+
+    public TrackRemoteDataSource() {
+    }
+
+    public static TrackRemoteDataSource getInstance() {
+        return sTrackRemoteDataSource;
+    }
+
+    public static void init() {
+        if (sTrackRemoteDataSource == null) {
+            sTrackRemoteDataSource = new TrackRemoteDataSource();
+        }
+    }
+
+    @Override
+    public void getListTrack(String url, String genre, int limit, int offSet,
+                             final Callback<List<Track>> callback) {
+        new LoadAsync(new TrackDataSource.Callback<String>() {
+            @Override
+            public void onStartLoading() {
+                callback.onStartLoading();
+            }
+
+            @Override
+            public void onGetSuccess(String data) {
+                if (data == null) {
+                    callback.onGetFailure(Constant.ERROR_NULL);
+                    return;
+                }
```

Comment:
ERROR: '}' at column 17 should be on the same line as the next part of a multi-block statement (one that directly contains multiple blocks: if/else-if/else, do/while or try/catch/finally).